backend/sirius-web-emf/src/main/java/org/eclipse/sirius/web/emf/services/EObjectIDManager.java

```diff
@@ -92,7 +93,9 @@ public String getOrCreateId(EObject eObject) {
     public void setId(EObject eObject, String id) {
         this.clearId(eObject);
 
-        eObject.eAdapters().add(new IDAdapter(UUID.fromString(id)));
+        if (eObject != null) {
+            eObject.eAdapters().add(new IDAdapter(UUID.fromString(id)));
+        }
```

Comment:
Not sure this part is necessary if the other part in DocumentService configures the ResourceSet correctly.

Reply:
Done,
I also removed the other changes in the same file as it concerns the same fix.